server/data-service/src/main/java/edu/illinois/ncsa/incore/service/data/utils/FileUtils.java

```diff
@@ -35,6 +42,11 @@
 import java.util.*;
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipOutputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
 
```

Comment:
These 5 sql packages are not used. Maybe good to remove?

Reply:
removed